nhsuk-design-system/views/includes/header-v1/header.html

```diff
@@ -41,6 +41,97 @@
 } );
 </script>
 
+{% if cookieBanner == 'active' %}
+<script>
+  function hideCookieBanner() {
+      var cookiebanner = document.getElementById("cookiebanner");
+      var cookiebannerstart = document.getElementById("cookiebanner-info");
+      var cookiebannerend = document.getElementById("cookiebanner-close");
+      cookiebannerstart.style.display = "none";
+      cookiebannerend.style.display = "block";
+  }
+
+  function showToggle() {
+    var element = document.getElementById("panel");
+    element.classList.toggle("open");
+  }
+</script>
+
+<div class="nhsuk-cookie-banner" id="cookiebanner" lang="[#LANGUAGE#]" dir="[#TEXTDIRECTION#]" ng-non-bindable>
```

Comment:
Does this need the ng-non-bindable attribute?

Reply:
This is required for Cookiebot functionality so left that in.